deploy/operator/base/rbac.yaml

```diff
@@ -22,6 +24,15 @@ rules:
       - update
       - watch
 
+  - apiGroups:
+      - quota.openshift.io
+    resources:
+      - clusterresourcequotas
```

Comment:
I might investigate  if it is possible to install this CRD into non-openshift clusters. Otherwise this will make the build service openshift only.

Reply:
this will be changing eventually to kcp cluster scoped quota
I based off of what staging cluster was doing when I started https://github.com/codeready-toolchain/host-operator/blob/master/deploy/templates/nstemplatetiers/appstudio/cluster.yaml#L15